paasta_tools/paasta_cli/cmds/logs.py

```diff
@@ -510,7 +510,7 @@ def tail_paasta_logs(service, levels, components, clusters, raw_mode=False):
 def paasta_logs(args):
     """Print the logs for as Paasta service.
     :param args: argparse.Namespace obj created from sys.args by paasta_cli"""
-    if 'scribereader' not in vars():
+    if scribereader is None:
```

Comment:
Much less scary.

However, still lacking automated tests; might be time to start considering those for this functionality.